.github/workflows/ci.yml

```diff
@@ -4,10 +4,7 @@ on:
   push:
     branches:
       - main
-  pull_request:
-    branches:
-      - main
-      - runners-1.0
+  pull_request
```

Comment:
```suggestion
  pull_request:
```

Reply:
I think we want it to just be a string if we're not configuring it.

Reply:
can you link it to the action documentation, i can't seem to find it

Reply:
iirc 
```yaml
on: [push, pull_request]
``` would do the trick
so what you are having is probably right.